src/platform/openbsd.rs

```diff
@@ -188,6 +248,45 @@ impl PlatformMemory {
     }
 }
 
+fn statfs_to_fs(fs: &statfs) -> Filesystem {
+    Filesystem {
+        files: (fs.f_files as usize).saturating_sub(fs.f_ffree as usize),
+        files_total: fs.f_files as usize,
+        files_avail: fs.f_ffree as usize,
+        free: ByteSize::b(fs.f_bfree * fs.f_bsize as u64),
+        avail: ByteSize::b(fs.f_bavail as u64 * fs.f_bsize as u64),
+        total: ByteSize::b(fs.f_blocks * fs.f_bsize as u64),
+        name_max: fs.f_namemax as usize,
+        fs_type: unsafe { ffi::CStr::from_ptr(&fs.f_fstypename[0]).to_string_lossy().into_owned() },
+        fs_mounted_from: unsafe { ffi::CStr::from_ptr(&fs.f_mntfromname[0]).to_string_lossy().into_owned() },
+        fs_mounted_on: unsafe { ffi::CStr::from_ptr(&fs.f_mntonname[0]).to_string_lossy().into_owned() },
+    }
+}
+
+#[repr(C)]
+#[derive(Debug, Clone, Copy)]
+// Fields of KERN_CPTIME2: https://github.com/openbsd/src/blob/master/sys/sys/sched.h#L83
```

Comment:
Maybe make it a doc comment `///` and make it a permalink to a particular commit (e.g. the latest release tag) instead of to `master`?

Reply:
Ok